### DISCUSSION TITLE
Train Conformer-CTC ASR model on character tokenzier

### DISCUSSION THREAD

**Discussion by neso613** [NVIDIA-NeMo/NeMo#7202] [General]
Following this [doc](https://github.com/NVIDIA/NeMo/blob/stable/tutorials/asr/ASR_CTC_Language_Finetuning.ipynb) to fine-tune conformer-ctc model.

In this, subword tokenzier used for conformer-ctc and character tokenzier used for quartznet.

can we use character tokenzier for conformer-ctc model?
If yes, how?

**Comment by titu1994** [comment]
You can try, but it will probably not be very stable so you'll need to update hyper params. We don't experiment with char models unless they are for Asian languages with large vocabularies

**Comment by bene-ges** [comment]
@neso613, @titu1994 I recently tried this [nemo config](https://github.com/NVIDIA/NeMo/blob/main/examples/asr/conf/conformer/conformer_ctc_char.yaml) for training char conformer-ctc from scratch. It works ok as it is, fast, stable and good quality.

**Reply by titu1994** [reply]
That's good news ! Any changes to LR or weight decay ?

**Reply by bene-ges** [reply]
no changes, just default config
the simplest test for English was on an4 dataset,
for Russian language I used a larger dataset, but just replaced the alphabet
